appveyor-ubuntu.yml

```diff
@@ -74,7 +74,7 @@ install:
   - sh: "JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64"
   - sh: "PATH=$JAVA_HOME/bin:$PATH"
   - sh: "source ${HOME}/venv${PYTHON_VERSION}/bin/activate"
-  - sh: "rvm use 2.5"
```

Comment:
We can remove support for building `ruby2.7` functions as well. (not in this one but in follow-up PR)